iceoryx_hoofs/test/integrationtests/test_resizeable_lockfree_queue_stresstest.cpp

```diff
@@ -45,11 +46,58 @@ struct Data
     uint64_t count{0};
 };
 
+class Barrier
```

Comment:
Could you move this into testing? I'm sure we could make use of this in other tests as well.

Reply:
Can do, but it was just a quick draft to try whether this is useful. Will move later but I do not think we want this in 2.0 do we?

Reply:
I don't think it's necessary for v2. This problem only occurs under high load and if really necessary, we could still backport to the v2 branch.